CmpApiModel missing in index.ts

**Version**
1.5.5

**Module (cmpapi, cli, stub, or testing)**
cmpapi

**Describe with reproduction steps – What is the expected behavior?**
Unresolved dependencies

When i change the CmpApiModel path (used with version 1.3.0) from

`import { CmpApiModel } from '@iabtcf/cmpapi/lib/CmpApiModel';` to
`import { CmpApiModel } from "@iabtcf/cmpapi/lib/mjs/CmpApiModel";`

then there are warnings from rollupjs module bundler
`(!) Plugin node-resolve: Could not resolve import "@iabtcf/cmpapi/lib/mjs/CmpApiModel" `

and webpack 
`Module not found: Error: Package path ./lib/mjs/CmpApiModel is not exported from package /Users/john/workspace/example/node_modules/@iabtcf/cmpapi (see exports field in /Users/john/workspace/example/node_modules/@iabtcf/cmpapi/package.json)
`

Is there a reason why the class is not exported? Is it possible to add CmpApiModel to the index.ts?

--- a/modules/cmpapi/src/index.ts
+++ b/modules/cmpapi/src/index.ts
@@ -2,5 +2,6 @@ export * from './command/index.js';
 export * from './response/index.js';
 export * from './status/index.js';
 export * from './CmpApi.js';
+export * from './CmpApiModel.js';
 export * from './CustomCommands.js';
 export {API_KEY, APIArgs} from './CallResponder.js';

